playbooks/vars/oci/profiles/ha/network.yaml

```diff
@@ -0,0 +1,332 @@
+################################
+# AWS Networks
+# AWS us-east-1: 10.0.0.0/16 (to 10.0.255.255/16)
+# AWS <unassigned>: 10.23.0.0/16 (to 10.23.255.255/19)
+
+#########################
+
+# TODO: fix those rules to more restrictive. This is used to dev env.
+# security_groups: []
+
+cloud_networks:
+  ## OCI US San Jose 1 (HA topology)
+  - name: "{{ cluster_state.infra_id }}-vcn"
+    block: "{{ okd_net_default_cidr }}"
+    provider: oci
+    region: "{{ config_cluster_region }}"
+    compartment_id: "{{ oci_compartment_id }}"
+
+    security_groups: "{{ security_groups | d([]) }}"
+    tags: "{{ cluster_state.tags | d({}) }}"
+
+    spec_vcn:
+      dns_label: ocp
+
+    internet_gateway:
+      spec:
+        is_enabled: yes
+        display_name: "{{ cluster_state.infra_id }}-igw"
+
+    nat_gateway:
```

Comment:
AWS supports many NGWs by VPC, the model of OCI subnet works (regional), multiple gateways by VCN seems to be useless.